registry/resolver.go

```diff
@@ -92,7 +92,7 @@ func (rr *registryResolver) updateAddressesFromRegistry() {
 		Addresses: addrs,
 	})
 	if err != nil {
-		logger.Errorln("[RESOLVER] service connection update failed")
+		logger.Errorf("[RESOLVER] service connection update failed from registry: `%v`, err: %v", rr.service.Name(), err)
```

Comment:
You forgot a new-line, which is implicitly added at the end of the string by `logger.Errorln`:

```suggestion
		logger.Errorf("[RESOLVER] service connection update failed from registry: `%v`, err: %v\n", rr.service.Name(), err)
```

Reply:
Looking at the log output in other parts of the file, I noticed that when referring to a service, the name is taken from `rr.service.Name()`, however when referring to a registry, it's `is`.

Applying the same logic to this line, I end up with this:

```go
logger.Errorf("[RESOLVER] service (`%v`) connection update failed from registry `%v`: %v\n", rr.service.Name(), is, err)
```

I didn't test it, though.

Reply:
The output looks like this 
```
ERROR: [registry_resolver] [RESOLVER] service (`search`) connection update failed from registry `[]`: bad resolver state
```
It looks horrible and the `is` refers for example to current empty `[]` configuration that is incorrect. However when we provide configuration with invalid hosts e.g `["web.example.co"]` you have to manually check and reconfigure it in yaml file.

I will change the order but i will strip `is` from log. I dont see benefit that it would be logging into error output.